src/thing/get.test.ts

```diff
@@ -1139,7 +1141,97 @@ describe("getStringWithLocale", () => {
   });
 });
 
-describe("getStringsWithLocaleAll", () => {
+describe("getStringByLocale", () => {
+  function makeQuad(
+    subject: string,
+    predicate: string,
+    value: string,
+    locale: string | NamedNode | undefined
```

Comment:
We have aliases for these :) 

```suggestion
    subject: IriString,
    predicate: IriString,
    value: IriString,
    locale?: IriString | Iri
```

Reply:
Also, it looks like `locale` is actually `localeOrLiteralType`?